lib/graphql/schema.rb

```diff
@@ -544,7 +568,11 @@ def rebuild_artifacts
         @rebuilding_artifacts = true
         traversal = Traversal.new(self)
         @types = traversal.type_map
+        @root_types = [query, mutation, subscription]
         @instrumented_field_map = traversal.instrumented_field_map
+        @type_reference_map = traversal.type_reference_map
+        @unions = traversal.unions
```

Comment:
is this ivar now unused? can we remove it (Traversal#unions)?

Reply:
yup they're both useless now! ✂️